src/NServiceBus.Core.Analyzer/Sagas/SagaDiagnostics.cs

```diff
@@ -6,63 +6,65 @@ static class SagaDiagnostics
     {
         const string DiagnosticCategory = "NServiceBus.Sagas";
 
-        public static readonly DiagnosticDescriptor NonMappingExpressionUsedInConfigureHowToFindSaga = new DiagnosticDescriptor(
+        public static readonly DiagnosticDescriptor NonMappingExpressionUsedInConfigureHowToFindSaga = new(
             id: DiagnosticIds.NonMappingExpressionUsedInConfigureHowToFindSaga,
             title: "Non-mapping expression used in ConfigureHowToFindSaga method",
             messageFormat: "The ConfigureHowToFindSaga method should only contain mapping expressions (i.e. 'mapper.MapSaga().ToMessage<T>()') and not contain any other logic.",
             category: DiagnosticCategory,
             defaultSeverity: DiagnosticSeverity.Error,
             isEnabledByDefault: true);
 
-        public static readonly DiagnosticDescriptor SagaMappingExpressionCanBeSimplified = new DiagnosticDescriptor(
+        public static readonly DiagnosticDescriptor SagaMappingExpressionCanBeSimplified = new(
             id: DiagnosticIds.SagaMappingExpressionCanBeSimplified,
             title: "Saga mapping expressions can be simplified",
             messageFormat: "The saga mapping contains multiple .ToSaga(…) expressions which can be simplified using mapper.MapSaga(…).ToMessage<T>(…) syntax.",
             category: DiagnosticCategory,
             defaultSeverity: DiagnosticSeverity.Warning,
             isEnabledByDefault: true);
 
-        public static readonly DiagnosticDescriptor MultipleCorrelationIdValues = new DiagnosticDescriptor(
+        public static readonly DiagnosticDescriptor MultipleCorrelationIdValues = new(
             id: DiagnosticIds.MultipleCorrelationIdValues,
             title: "Saga can only define a single correlation property on the saga data",
             messageFormat: "The saga can only map the correlation ID to one property on the saga data class.",
             category: DiagnosticCategory,
             defaultSeverity: DiagnosticSeverity.Error,
             isEnabledByDefault: true);
 
-        public static readonly DiagnosticDescriptor MessageStartsSagaButNoMapping = new DiagnosticDescriptor(
+        public static readonly DiagnosticDescriptor MessageStartsSagaButNoMapping = new(
             id: DiagnosticIds.MessageStartsSagaButNoMapping,
             title: "Message that starts the saga does not have a message mapping",
-            messageFormat: "Saga {0} implements IAmStartedByMessages<{1}> but does not provide a mapping for that message type. In the ConfigureHowToFindSaga method, after calling mapper.MapSaga(saga => saga.CorrelationPropertyName), add .ToMessage<{1}>(msg => msg.PropertyName) to map a message property to the saga correlation ID, or .ToMessageHeader<{1}>(\"HeaderName\") to map a header value that will contain the correlation ID.",
+            messageFormat: @"Saga {0} implements IAmStartedByMessages<{1}> but does not provide a mapping for that message type. 
```

Comment:
Intentionally not mentioning the specifics of the advanced mapping tecniques to guide towards the thing that should likely be used